<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="DirectoryTreeView" name="treeView">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="tabletTracking">
        <bool>true</bool>
       </property>
       <attribute name="headerMinimumSectionSize">
        <number>39</number>
       </attribute>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="TextEditTab" name="tabWidget">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
       <widget class="QTextBrowser" name="logWindow"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionAdd_new_note"/>
    <addaction name="actionOpen_file"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionLog"/>
    <addaction name="actionWord_Wrap"/>
   </widget>
   <widget class="QMenu" name="menuGoto">
    <property name="title">
     <string>Goto</string>
    </property>
    <addaction name="actionGo_Back"/>
    <addaction name="actionJump_Forward"/>
   </widget>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>Preferences</string>
    </property>
    <widget class="QMenu" name="menuFont">
     <property name="title">
      <string>Font</string>
     </property>
     <addaction name="actionSmaller"/>
     <addaction name="actionLarger"/>
     <addaction name="separator"/>
     <addaction name="actionReset"/>
    </widget>
    <widget class="QMenu" name="menuChange_Color_Sheme">
     <property name="title">
      <string>Change Color Sheme</string>
     </property>
     <addaction name="actionDark"/>
     <addaction name="actionDefault"/>
     <addaction name="actionGrey"/>
     <addaction name="actionDark_Blue"/>
    </widget>
    <addaction name="menuChange_Color_Sheme"/>
    <addaction name="menuFont"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuGoto"/>
   <addaction name="menuPreferences"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_new_note">
   <property name="text">
    <string>Add new note</string>
   </property>
  </action>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="statusTip">
    <string>fewfer</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Log Window</string>
   </property>
  </action>
  <action name="actionWord_Wrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Word Wrap</string>
   </property>
  </action>
  <action name="actionGo_Back">
   <property name="text">
    <string>Jump Back</string>
   </property>
  </action>
  <action name="actionJump_Forward">
   <property name="text">
    <string>Jump Forward</string>
   </property>
  </action>
  <action name="actionSmaller">
   <property name="text">
    <string>Smaller</string>
   </property>
  </action>
  <action name="actionLarger">
   <property name="text">
    <string>Larger</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionGrey">
   <property name="text">
    <string>Grey</string>
   </property>
  </action>
  <action name="actionDark_Blue">
   <property name="text">
    <string>Dark Blue</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DirectoryTreeView</class>
   <extends>QTreeView</extends>
   <header>directorytreeview.h</header>
  </customwidget>
  <customwidget>
   <class>TextEditTab</class>
   <extends>QTabWidget</extends>
   <header>textedittab.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
